.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ commands:
 jobs:
   test-package:
     docker:
-      - image: mozilla/fxa-circleci
```

Comment:
Why the image switch? (I might figure this out further through the PR)

Reply:
This is a case of me noticing that we don't need a special image here while I was making the other change. The circleci images tend to load faster than custom ones.